object-screenshotter.js

```diff
@@ -25,6 +25,7 @@ export const screenshotObjectUrl = async ({
   start_url,
   width = 300,
   height = 300,
+  canvas

```

Comment:
Why pass the canvas? We already set the size on the internal one.

Reply:
discussed this in discord and added checks.
if canvas passed width/height should not be.